### PR TITLE
Prevent crash when plot window is docked as document window and hidden

### PR DESCRIPTION
**Pull request by huguesv**
Fixes https://github.com/Microsoft/RTVS/issues/2375

### CODE REVIEW

File: src/R/Components/Impl/Plots/Implementation/NonScaledImage.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ internal class NonScaledImage : Image {
                 // We prevent that scaling by calculating a size based on
                 // pixel size and dpi setting.
                 Size bitmapSize = new Size(bmp.PixelWidth, bmp.PixelHeight);
-                return WpfUnitsConversion.FromPixels(this, bitmapSize);
+                return WpfUnitsConversion.FromPixels(PresentationSource.FromVisual(this), bitmapSize);
```
Comment (MikhailArkhipov) [inline src/R/Components/Impl/Plots/Implementation/NonScaledImage.cs]: So, according to below, 
```
var source = PresentationSource.FromVisual(Content as Visual);
if (source != null) {
```
PresentationSource.FromVisual can still return null?

Comment (MikhailArkhipov) [inline src/R/Components/Impl/Plots/Implementation/NonScaledImage.cs]: Or it never does that for `this` ?

Comment (huguesv) [inline src/R/Components/Impl/Plots/Implementation/NonScaledImage.cs]: It's never null here.

**Comment by MikhailArkhipov** [comment]
👍

**Comment by MikhailArkhipov** [comment]
Let me know when merged to 0_5, i can start the build

**Pull request merged**